Explicitly add missing Qt5Netowrk deps.
Fixes:
In file included from /var/tmp/portage/gui-apps/paperde-0.1.1_p20210426/work/paperde-15018307192107e2915671d412f3fc900ac5af4b/papershell/main.cpp:34:
/usr/include/cprime/capplication.h:39:10: fatal error: QLocalServer: No such file or directory
   39 | #include <QLocalServer>
      |          ^~~~~~~~~~~~~~
--- a/papershell/CMakeLists.txt
+++ b/papershell/CMakeLists.txt
@@ -25,6 +25,7 @@ set( CMAKE_AUTOUIC ON )
 
 find_package( Qt5Core REQUIRED )
 find_package( Qt5Gui REQUIRED )
+find_package( Qt5Network REQUIRED )
 find_package( Qt5Widgets REQUIRED )
 find_package( Qt5DBus REQUIRED )
 find_package( dbusmenu-qt5 REQUIRED )
@@ -121,7 +122,7 @@ set ( RESOURCES
 add_executable( papershell ${SOURCES} ${HEADERS} ${UIS} ${RESOURCES}  )
 target_link_libraries(
     papershell
-    Qt5::Core Qt5::Gui Qt5::Widgets Qt5::DBus           # Qt5 Libs
+    Qt5::Core Qt5::Gui Qt5::Network Qt5::Widgets Qt5::DBus           # Qt5 Libs
     dbusmenu-qt5                                        # DBusMenu Qt5 for tray menu
     cprime-widgets cprime-gui cprime-core               # CuboCore Libs
     paperdegui paperdecore paperdewl                    # PaperDE Libs
--- a/sessionmanager/CMakeLists.txt
+++ b/sessionmanager/CMakeLists.txt
@@ -25,6 +25,7 @@ set( CMAKE_AUTOUIC ON )
 
 find_package( Qt5Core REQUIRED )
 find_package( Qt5DBus REQUIRED )
+find_package( Qt5Network REQUIRED )
 
 set( HEADERS
 	sessionmgr.h
@@ -38,7 +39,7 @@ set( SOURCES
 )
 
 add_executable( papersessionmanager ${SOURCES} ${HEADERS}  )
-target_link_libraries( papersessionmanager Qt5::Core Qt5::DBus cprime-core csys paperdecore )
+target_link_libraries( papersessionmanager Qt5::Core Qt5::DBus Qt5::Network cprime-core csys paperdecore )
 
 install( TARGETS papersessionmanager DESTINATION bin )
 install( FILES paperdesktop.desktop DESTINATION share/wayland-sessions/ )
--- a/settings/CMakeLists.txt
+++ b/settings/CMakeLists.txt
@@ -25,6 +25,7 @@ set( CMAKE_AUTOUIC ON )
 
 find_package( Qt5Core REQUIRED )
 find_package( Qt5Gui REQUIRED )
+find_package( Qt5Network REQUIRED )
 find_package( Qt5Widgets REQUIRED )
 find_package( Qt5UiTools REQUIRED )
 
@@ -50,7 +51,7 @@ set ( RESOURCES
 add_executable( papersettings ${SOURCES} ${HEADERS} ${UIS} ${RESOURCES}  )
 target_link_libraries(
     papersettings
-    Qt5::Core Qt5::Gui Qt5::Widgets Qt5::UiTools
+    Qt5::Core Qt5::Gui Qt5::Network Qt5::Widgets Qt5::UiTools
     cprime-widgets cprime-core
     paperdegui paperdecore paperdewl
 )
